src/include/detail/flat/qv.h

```diff
@@ -134,15 +141,22 @@ auto qv_query_heap(
 }
 
 // @todo Add to out of core
-template <class T, class DB, class Q, class ID>
+template <
+    class T,
+    feature_vector_array DB,
+    feature_vector_array Q,
+    class ID,
+    class Distance = sum_of_squares_distance>
 auto qv_query_heap(
     T,
     const DB& db,
     const Q& query,
     const ID& ids,
     int k_nn,
-    unsigned nthreads) {
+    unsigned nthreads,
+    Distance distance = Distance{}) {
   scoped_timer _{tdb_func__};
+  // load(db);
```

Comment:
(Minor) What's going on here? Should we load? If not, let's remove this?

Reply:
This touches on an issue we need to clean up:
https://app.shortcut.com/tiledb-inc/story/40229/specify-definitive-behavior-for-out-of-core-algorithms-and-data-structures